lib/dndtoc.sty

```diff
@@ -0,0 +1,27 @@
+\ExplSyntaxOn
+\RequirePackage{titletoc}
+
+% Makes the TOC go to section depth
+\setcounter{tocdepth}{1}
+
+\titlecontents{part}
+  [0pt] % left
+  {\DndFontTocPart} % above-code
+  {\partname\ \thecontentslabel :~ } % numbered-entry-format
+  {} % numberless-entry-format
+  {} % filler-page-format
+  [ \vspace{-.3ex} \titleline{ \color{titlegold} \titlerule[1pt] }\vspace{.3ex} ] % below-code
+
+\titlecontents{chapter}
+  [0pt] % left
+  {\DndFontTocChapter} % above-code
+  {Ch.\ \thecontentslabel :~ } % numbered-entry-format
```

Comment:
"Ch." needs to be added to the captions files. We cannot emit English text directly.